core/trino-spi/src/main/java/io/trino/spi/StandardErrorCode.java

```diff
@@ -160,6 +160,7 @@ public enum StandardErrorCode
     INVALID_BRANCH_PROPERTY(136, USER_ERROR),
     INVALID_DEFAULT_COLUMN_VALUE(137, USER_ERROR),
     INVALID_GRACE_PERIOD(138, USER_ERROR),
+    INVALID_INTERVAL_QUALIFIER(139, USER_ERROR),
```

Comment:
To me this could also be a SYNTAX_ERROR perhaps?

Reply:
Fair. Removed.